std/experimental/color/rgb.d

```diff
@@ -0,0 +1,852 @@
+module std.experimental.color.rgb;
+
+import std.experimental.color;
+import std.experimental.color.xyz;
+
+import std.traits: isInstanceOf, isNumeric, isIntegral, isFloatingPoint, isSigned, isSomeChar, Unqual;
+import std.typetuple: TypeTuple;
+import std.typecons: tuple;
+
+@safe: pure: nothrow: @nogc:
+
+
+enum isRGB(T) = isInstanceOf!(RGB, T);
+
+enum RGBColorSpace
+{
+    sRGB,
+    sRGB_Gamma2_2,
+/+ TODO: these all need testing
```

Comment:
> There is a linear version of every colour space, so it seemed silly to include them in this table.

Is it so? But the "linear versions" of sRGB and RGB with whatever gamma are the same. RGB with gamma 1.0 is linear